OpenXmlPowerTools/HtmlToWmlCssParser.cs

```diff
@@ -1306,33 +1093,19 @@ public bool IsColor
                         return false;
                     }
 
-                    if (ColorParser.IsValidName(m_val))
+                    if (ColorParser.IsValidName(Value))
                     {
                         return true;
                     }
                 }
-                else if (m_type == CssTermType.Function)
+                else if (Type == CssTermType.Function)
                 {
-                    if ((m_function.Name.ToLower().Equals("rgb") && m_function.Expression.Terms.Count == 3)
-                        || (m_function.Name.ToLower().Equals("rgba") && m_function.Expression.Terms.Count == 4)
-                        )
-                    {
-                        for (var i = 0; i < m_function.Expression.Terms.Count; i++)
-                        {
-                            if (m_function.Expression.Terms[i].Type != CssTermType.Number)
-                            {
-                                return false;
-                            }
-                        }
-                        return true;
-                    }
-                    else if ((m_function.Name.ToLower().Equals("hsl") && m_function.Expression.Terms.Count == 3)
-                      || (m_function.Name.ToLower().Equals("hsla") && m_function.Expression.Terms.Count == 4)
-                      )
+                    if (((Function.Name.ToLower().Equals("hsl") || Function.Name.ToLower().Equals("rgb")) && Function.Expression.Terms.Count == 3) ||
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)